packages/plugin-ext/src/main/browser/view/view-context-key-service.ts

```diff
@@ -35,7 +35,7 @@ export class ViewContextKeyService {
      * Viewlet is a tab in the left area in VS Code. Active means visible in this context.
      *
      * In VS Code there can be only one visible viewlet at any time.
-     * It is not true for Theia, since views can be relayouted to different areas.
+     * It is not true for Theia, since views can be relocated to different areas.
```

Comment:
not completely sure about this one... `relayouted` though suspicious sounds like it could mean something like "layed-out again". Which makes more sense, that or `relocated`?

Reply:
I updated it to be more explicit and wrote 'layed-out again to different areas'.